bridges/chains/chain-asset-hub-westend/Cargo.toml

```
@@ -19,10 +19,16 @@ scale-info = { features = ["derive"], workspace = true }
 
 # Substrate Dependencies
 frame-support = { workspace = true }
+sp-api = { workspace = true }
 sp-core = { workspace = true }
+sp-runtime = { workspace = true }
+sp-std = { workspace = true }
```

I think we don't need this deps here, right?

```suggestion
sp-core = { workspace = true }
```